DiceKit/ProbabilityMass.swift

```diff
@@ -8,28 +8,36 @@
 
 import Foundation
 
-public struct ProbabilityMass: ApproximatelyEquatable {
+// TODO: Find a proper place for this
+public struct ProbabilityMassConfig {
```

Comment:
This has to be extracted because you can't have stored properties on generic types. Still not sure where it should live. I'm open to suggestions (even if it's "right where it's at").